configuration/monitor-acs.adoc

```diff
@@ -1,13 +1,47 @@
 [id="monitor-acs"]
-= Monitoring with Prometheus
+= Monitoring {product-title-short}
 include::modules/common-attributes.adoc[]
 :context: monitor-acs
 
 toc::[]
 
+== Monitoring with {osp}
+
+[role="_abstract"]
+link:https://docs.openshift.com/container-platform/4.13/monitoring/monitoring-overview.html[{ocp}] includes a preconfigured,
+preinstalled, and self-updating monitoring stack that provides monitoring for core platform components.
+{product-title-short} has support for {osp} monitoring by using an encrypted and authenticated endpoint accessible only by the Prometheus Operator.
```

Comment:
```suggestion
{product-title-short} exposes metrics to {osp} monitoring via an encrypted and authenticated endpoint.
```

While OpenShift monitoring uses Prometheus operator under the hood, it is an implementation detail. Also it's the Prometheus pod, not the Prometheus operator pod that scrapes the metrics.